lib/livebook/hubs/enterprise_client.ex

```diff
@@ -86,6 +86,14 @@ defmodule Livebook.Hubs.EnterpriseClient do
     {:ok, %__MODULE__{hub: enterprise, server: pid}}
   end
 
+  @impl true
+  def handle_continue(:create_session, state) do
+    data = LivebookProto.build_session_request(app_version: Livebook.Config.app_version())
+    {:session, _} = ClientConnection.send_request(state.server, data)
```

Comment:
I've just realised that calling this `session` may cause confusion going forward, since we already have a `session` and it play a central role in the app. I can't think of a clear alternative, but if anyone does, shoot!

Reply:
We also could have the `session_created` event when it creates a Livebook Session. I agree we should change the name. I'm open to suggestions.

Reply:
So, in this case, it will be:

Topic: `:user_synchronized`
Event: `%UserSynchronized{}`
Key: `:sync`
Request: `%SyncRequest{}`
Response: `%SyncResponse{}`

WDYT?

Reply:
Ah, if we do it initially, could also be `HandshakeRequest`, `HandshakeResponse`. Either works for me though!

What is `:user_synchronized`?

Reply:
> What is `:user_synchronized`?

The event `enterprise` sends when returning the `SessionResponse`.

> When it comes to topic+event, we probably should not have a new topic+event. Instead, we should emit the usual "session_created", "session_delete" and so on. WDYT?

We don't have this control on the other side, but we could.

> So for example, we compare the secrets that we got with the secrets that we have on the state (we will start with an empty list), and then emit the relevant events.

With this PR, we aren't going to start with an empty list anymore.

But I dunno if I understood correctly what you mean.